OtripleS.Web.Api/Services/StudentGuardians/StudentGuardianService.cs

```diff
@@ -61,10 +61,12 @@ public partial class StudentGuardianService : IStudentGuardianService
 		});
 
 		public ValueTask<StudentGuardian> RetrieveStudentGuardianByIdAsync(Guid studentId, Guid guardianId) =>
-		TryCatch(() =>
+		TryCatch(async () =>
 		{
 			ValidateStudentGuardianIdIsNull(studentId, guardianId);
-			return this.storageBroker.SelectStudentGuardianByIdAsync(studentId, guardianId);
+			StudentGuardian storageStudentGuardian = await this.storageBroker.SelectStudentGuardianByIdAsync(studentId, guardianId);
```

Comment:
@AgentEnder look like too many characters in one line

Reply:
Fixed in latest commit